_includes/accessibility-tests/checklist-intros.html

```diff
@@ -0,0 +1,24 @@
+{:.border-top .padding-top-1 .border-gray-10}
+
+USWDS performed these tests to check the {{ component_name }}'s accessibility and usability for {{ AT_type | default: title | downcase }}.
+For help performing your own {{ title | downcase }} tests, check out the resources in the expandable section below.
+
+{% if title == "Screen reader" %}
+  {% assign title = "screen readers" %}
+{% endif %}
+<div class="usa-accordion measure-5 margin-bottom-2">
+  <h4 class="usa-accordion__heading">
+    <button type="button" class="usa-accordion__button" aria-controls="{{ title | slugify | append:'-how-to' }}">
+      How to test with {{ title | downcase}}
+    </button>
+  </h4>
+  <div id="{{ title | slugify | append:'-how-to'}}" class="usa-accordion__content usa-prose border-1px border-gray-5 border-top-none">
+    {% if title == "Zoom magnification" %}
+      {% include accessibility-tests/how-to-zoom.html %}
+    {% elsif title == "screen readers" %}
```

Comment:
This has evolved over time. I needed to adjust "screen reader" to fit into the "How to test with [test type]" format. This was the mechanism to do that. I updated the code a bit in [fd200db](https://github.com/uswds/uswds-site/pull/2138/commits/fd200dbdcf4be71025708a2056b8ca01447234e8) to try to make it clearer. 

FWIW I got hints that we probably will be editing the copy in these sections in the future, so it might not be as formulaic moving forward.